st2reactor/st2reactor/rules/enforcer.py

```diff
@@ -103,6 +118,13 @@ def _update_trace(self):
                                                               rules=[str(self.rule.id)])
```

Comment:
Note to self : Should save rule_enforcement_id instead of rule.id. I will take care of this separately